site/overview/faqs.md

```diff
@@ -0,0 +1,154 @@
+  - - meta
+    - name: description
+      content: Learn how to get started with Alchemy's Account Kit, a vertically integrated stack for building apps that support ERC-4337.
+  - - meta
+    - property: og:description
+      content: Frequently Asked Questions
+  - - meta
+    - name: twitter:title
+      content: FAQs
+  - - meta
+    - name: twitter:description
+      content: Frequently Asked Questions
+---
+
+# Frequently Asked Questions
+
+## Smart Contract Accounts - LightAccount
+
+### Do accounts have the same address across all chains?
+
+::: details Answer
+In almost all cases, yes, you will get the same address on all chains as long as the owner address is the same! The deployment address is a function of the owner address, the account implementation (e.g. latest version of LightAccount), and the salt (you can optionally specify this). If all three of those remain the same, then you deploy at the same contract address.
+
+There are two scenarios where you'd get a different contract address:
+
+1. If you deploy one account, then change the owner address, then deploy the second account
+2. If you upgrade the account (e.g. to a new version of LightAccount). It is unlikely that we will make many updates to this contract so the address will not change frequently.
+   :::
+
+### How would Alchemy initiate an upgrade of a LightAccount?
+
+::: details Answer
+It is unlikely we will frequently update the LightAccount contract itself, however it is possible if needed. LightAccount has [`UUPSUpgradeable`](https://github.com/alchemyplatform/light-account/blob/main/src/LightAccount.sol#L50) which adds upgrade methods on the account itself. To upgrade an account you will need to send a UO using that method.
+:::
+
+### Can I have multiple accounts for the same owner address? / How do I set the value of the salt/index for LightAccount?
+
+::: details Answer
+Yes! The optional index value (salt) on LightAccount enables the ability to have multiple accounts for the same owner address. This value defaults to 0. You can set it in the [constructor](/packages/aa-accounts/light-account/constructor.html#params-simplesmartaccountparams).
+:::
+
+## Submitting User Operations
+
+### How does the speed of user operations compare to normal transactions?
+
+::: details Answer
+If the User Operation (meta-transaction for 4337 accounts) is correctly priced and submitted a few hundred ms prior to a new block getting created, it will typically get placed in the next block. This is because the bundler needs time to create/propagate its transaction. You can think of it as 1 extra block time worth of latency, but we are working towards improving this latency.
+:::
+
+### Why am I seeing a delay in landing userOps on-chain?
+
+::: details Answer
+This can happen when UserOperations become underpriced, frequently due to fee market movement between when gas and fees are estimations and when the user operation is actually submitted.
```

Comment:
```suggestion
This can happen when `UserOperation`s become underpriced, frequently due to fee market movement between when gas and fees are estimations and when the user operation is actually submitted.
```